lms/static/js/student_account/enrollment.js

```diff
@@ -52,7 +60,58 @@
                     }
                 });
             },
+            /**
+             * Show a message in the frontend.
+             * @param  {Object} message The message to display.
+             * @param  {string} redirectUrl The URL to redirect to when the button is clicked.
+             */
+            showMessage: function(message, redirectUrl) {
+                const componentId = 'student-enrrollment-feedback-error';
```

Comment:
```suggestion
                const componentId = 'student-enrollment-feedback-error';
```
Should be my typo sorry 😅